cmd/boulder-wfe2/main.go

```diff
@@ -57,6 +66,97 @@ type config struct {
 	}
 }
 
+// loadChainFile loads a PEM certificate from the chainFile provided. It
+// validates that the PEM is well-formed with no leftover bytes, and contains
+// only a well-formed X509 certificate. If the chain file meets these
+// requirements the PEM bytes from the file are returned, otherwise an error is
+// returned.
+func loadChainFile(aiaIssuerURL, chainFile string) ([]byte, error) {
```

Comment:
nit: this name is slightly confusing, it's being used to construct a chain, but it's not reading a chain of certificates in a file, it's just reading a single certificate (otherwise it fails). Perhaps `loadCertificateFile` would be more appropriate.

Reply:
Good point! I can see how this would confuse.